test/functional/api/vim_spec.lua

```diff
@@ -2195,4 +2195,15 @@ describe('API', function()
       ]]}
     end)
   end)
+
+  describe('nvim_del_mark', function()
+    it('works',
```

Comment:
also test failure modes. e.g. what happens with lowercase marks? (I suggest returning a boolean, true=succeeded, false=failed)

Reply:
Thanks! I have changed the "deleting" functions signature to return a bool!